Nosenti/certificate-manager---Innocent
# Task10-KAN-41 Comments

#### PR description
- This PR is for adding comments based on the current user

#### Ticket / Jira Link
- [KAN-41](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-41)

#### Checklist:

#### How To Test Changes
- On the Certificate Form, click new comment
- The space to add a comment will show. 
- Add a comment and it should be show in list of comments with the current user
- Change the user in navigation bar and add another comment. The user attached to the comment will be the current user

#### Screenshots:
![image](https://github.com/user-attachments/assets/0d9402c5-2855-4d8d-91bb-9443f61e1850)

![image](https://github.com/user-attachments/assets/cac96bbb-626f-4863-bad6-c4f2137db701)
